File: SpectrumWeb/wwwroot/Local/ac_master.js
class ACMasterClass {
    PkRecordId;
    TransactId;
    TailNmbr;
    SerialNmbr;
    Description;
    DateAcquired;
    DateDispDuDef;
    DaysInService;
    DaysOutOfSvc;
    StatusState;
    MaintBase;
    AircraftType;
    AcIdnt;
    Date;
    Hours;
    CheckForRemvl;
    Cycles;
    Weeks;
    FrstClassSeats;
    BusClassSeats;
    CoachSeats;
    FltFollName;
    IncRelRpts;
    constructor(
        pkRecordId,
        transactId,
        tailNmbr,
        serialNmbr,
        description,
        dateAcquired,
        dateDispDuDef,
        daysInService,
        daysOutOfSvc,
        statusState,
        maintBase,
        aircraftType,
        acIdnt,
        date,
        hours,
        checkForRemvl,
        cycles,
        weeks,
        frstClassSeats,
        busClassSeats,
        coachSeats,
        fltFollName,
        incRelRpts
    ) {
        this.PkRecordId = pkRecordId;
        this.TransactId = transactId;
        this.TailNmbr = tailNmbr,
        this.SerialNmbr = serialNmbr,
        this.Description = description,
        this.DateAcquired = dateAcquired,
        this.DateDispDuDef = dateDispDuDef,
        this.DaysInService = daysInService,
        this.DaysOutOfSvc = daysOutOfSvc,
        this.StatusState = statusState,
        this.MaintBase = maintBase,
        this.AircraftType = aircraftType,
        this.AcIdnt = acIdnt,
        this.Date = date,
        this.Hours = hours,
        this.CheckForRemvl = checkForRemvl,
        this.Cycles = cycles,
        this.Weeks = weeks,
        this.FrstClassSeats = frstClassSeats,
        this.BusClassSeats = busClassSeats,
        this.CoachSeats = coachSeats,
        this.FltFollName = fltFollName,
        this.IncRelRpts = incRelRpts
    }
}

File: SpectrumWeb/Models/AcMaster.cs
﻿using System;
using System.Collections.Generic;

namespace SpectrumWeb.Models;

public partial class AcMaster
{
    public string PkRecordId { get; set; } = null!;

    public string PkTransactionId { get; set; } = null!;

    public string? TailNmbr { get; set; }

    public string? SerialNmbr { get; set; }

    public string? Description { get; set; }

    public DateOnly? DateAcquired { get; set; }

    public DateOnly? DateDispDuDef { get; set; }

    public int? DaysInService { get; set; }

    public int? DaysOutOfSvc { get; set; }

    public string? StatusState { get; set; }

    public string? MaintBase { get; set; }

    public string? AircraftType { get; set; }

    public string? AcIdnt { get; set; }

    public DateOnly? Date { get; set; }

    public double? Hours { get; set; }

    public bool? CheckForRemvl { get; set; }

    public int? Cycles { get; set; }

    public short? Weeks { get; set; }

    public short? FrstClassSeats { get; set; }

    public short? BusClassSeats { get; set; }

    public short? CoachSeats { get; set; }

    public string? FltFollName { get; set; }

    public bool? IncRelRpts { get; set; }
}
